root/static/scripts/release-editor/fields.js

```diff
@@ -876,6 +880,12 @@ class Release extends mbEntity.Release {
                 const isDuplicate = _.filter(events, nonEmptyEvent).length > 1;
                 events.forEach(e => e.isDuplicate(isDuplicate));
             });
+        }); 
+
+        this.earliestYear = ko.computed(function () {
+            return Math.min(
+            ...self.events().map(e => e.unwrapDate().year),
```

Comment:
This should be indented more (or perhaps it can all fit in the previous line)
```suggestion
                ...self.events().map(e => e.unwrapDate().year),
```